docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -514,7 +515,257 @@ If YugabyteDB Anywhere will be using **cron jobs**, make sure the yugabyte user
 YugabyteDB Anywhere **systemd services** to perform the monitoring operations mentioned above, then make sure ...
 -->
 
-You have finished configuring your on-premises cloud provider. Proceed to [Configure the backup target](../../backup-target/), or [Create deployments](../../../create-deployments/).
+You have finished configuring your on-premises cloud provider. Proceed to [Configure the backup target](../../backup-target/) or [Create deployments](../../../create-deployments/).
+
+##### Install systemd-related database service unit files
+
+As an alternative to setting crontab permissions, you can install systemd-specific database service unit files, as follows:
+
+1. Obtain sudo root access and add the `yugabyte` file to `/etc/sudoers.d/`. Next, open the `/etc/sudoers.d/yugabyte` file and write the following to provide permissions for the `yugabyte` user to run systemctl commands as a sudo user in order to manage the database:
```

Comment:
I think for this first step, we can just be a bit more general. We can say "Provider the yugabyte user the ability to run the following commands" and then have this shell block. Then, we can remove the specifications of /etc/sudoers.d/ and the includedir.

Reply:
Changed